command/plugin_info.go

```diff
@@ -114,6 +114,10 @@ func (c *PluginInfoCommand) Run(args []string) int {
 		"sha256":  resp.SHA256,
 	}
 
+	if resp.DeprecationStatus != "" {
```

Comment:
If DeprecationStatus is "", would it be more accurate for the response to be something like `"DeprecationStatus": "Unknown"`?

Also, is this check redundant given `omitempty`?

Reply:
Well, I believe `omitempty` is just for the json decoder. This is necessary to hide the `deprecation_status` field in `plugin info` output; however, I'm not entirely certain if that's the best behavior. This is one side-effect:

```
➜  vault-builtin-deprecation (d86bb29) ✗ /Users/mpalmi/scm/vault/__debug_bin plugin info -field deprecation_status auth app-id
Field "deprecation_status" not present in secret
```

As opposed to:

```
➜  vault-builtin-deprecation (d86bb29) ✗ /Users/mpalmi/scm/vault/__debug_bin plugin info -field deprecation_status auth app-id

```

I think I agree with having `""` -> `n/a`, considering the fact that fields like `sha256` behave like that way:
```
➜  vault-builtin-deprecation (d86bb29) ✗ /Users/mpalmi/scm/vault/__debug_bin plugin info auth app-id
Key                   Value
---                   -----
args                  []
builtin               true
command               n/a
deprecation_status    pending removal
name                  app-id
sha256                n/a
```

Reply:
Thanks @VioletHynes! I've pushed another commit to fix this.

Reply:
Perfect, thanks! That sums up everything so I'll resolve my comments.